examples/extractor/example_extractor.yml

```diff
@@ -16,10 +16,28 @@ source_repository: >-
     https://github.com/example/extractor
 documentation: >-
     https://example.github.io/extractor
-usage: >-
-    - cli:parse --type=example {{ file_path }}
-    - python:example.extrators.extract({{ file_path }}, type="example")
-    - python:example.extrators.extract.ExampleExtractor({{ file_path }}).parse()
+usage:
+    - method: cli
+      command: parse --type=example {{ file_path }}
+    - method: python
+      setup: example.extractors
+      command: extractors.extract({{ file_path }}, type="example")
+    - method: python
+      command: example.extractors.extract.ExampleExtractor({{ file_path }}).parse()
+installation:
+    - method: conda
+      requirements: >-
+          dependencies:
+            - python=3.4
+            - numpy
+            - toolz
+            - matplotlib
+            - dill
+            - pandas
+            - partd
+            - bokeh
+            - pip:
+              - git+https://github.com/blaze/dask.git#egg=dask[complete]
```

Comment:
Looking at the instructions below, would the equivalent `pip` example here just be:

```yaml
- method: pip
  packages: example-extractor
```

? I guess my only confusion would be that you could also just inline `example-extractor` as a requirement, so not sure I see the difference. I guess its really that one is intended to provide a full reproducible environment, in which case perhaps `environment` is a better name?